streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -760,13 +760,14 @@ public class StreamsConfig extends AbstractConfig {
 
     public static final String RACK_AWARE_ASSIGNMENT_STRATEGY_NONE = "none";
     public static final String RACK_AWARE_ASSIGNMENT_STRATEGY_MIN_TRAFFIC = "min_traffic";
+    public static final String RACK_AWARE_ASSIGNMENT_STRATEGY_BALANCE_SUBTOPOLOGY = "balance_subtopology";
```

Comment:
The naming does not include the goal of `minimum cross rack traffic` in the doc, should we name it to `RACK_AWARE_ASSIGNMENT_STRATEGY_MIN_TRAFFIC_BALANCE_SUBTOPOLOGY` instead?

Reply:
@ex172000 -- not sure if I can follow what you mean by your comment? Can you elaborate?